src/plot_recipes.jl

```diff
@@ -44,7 +44,7 @@ function _dataframe_plots_internal(
         @warn "Plot dataframe empty: skipping plot creation"
         p = Plots.plot!()
     else
-        data = convert(Matrix, no_datetime(variable))
+        data = Matrix(no_datetime(variable))
```

Comment:
Is `variable` a DataFrame here? I think there are some optimized methods in DataFrames.jl to make this conversion.